.github/workflows/test.yaml

```
@@ -4,17 +4,107 @@
+          path: dist
+
+  test:
+    name: Run ${{ matrix.tests.type }}
+    needs: [setup, build]
+    runs-on: [ubuntu-latest]
+    strategy:
+      fail-fast: false
+      matrix:
+        tests:
+          - type: Unit Tests
+            commandArgs: --file-path=${{ env.BASE_PATH }}/unit*.js --split=3 --parallel=3 --load-balance
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)